File: 프로그래머스/0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        String str = Integer.toString(order);
        str = str.replaceAll("0", "");
        for(int i = 0; i <= str.length() - 1; i++){
            if(str.charAt(i) % 3 == 0 && str.charAt(i) != 0 ) answer++;
        }
        return answer;
    }
}

File: 프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    let answer = 0;
    let str = order.toString();
    str = str.replace(/0/g, '');
    for (let i = 0; i < str.length; i++) {
        if (parseInt(str.charAt(i)) % 3 === 0 && str.charAt(i) !== '0') {
            answer++;
        }
    }
    return answer;
}
